Quantum-Control-Applications/Superconducting/Single Fixed Transmon/DRAG_calibration.py

```diff
@@ -86,11 +90,15 @@
 
     job = qm.execute(drag)
     res_handles = job.result_handles
-    I_handle = res_handles.get("I")
-    Q_handle = res_handles.get("Q")
+    I1_handle = res_handles.get("I1")
+    Q1_handle = res_handles.get("Q1")
+    I2_handle = res_handles.get("I2")
+    Q2_handle = res_handles.get("Q2")
     iteration_handle = res_handles.get("iteration")
```

Comment:
It is far from being mandatory, but you can try using the [fetching tool](https://github.com/qua-platform/py-qua-tools/tree/main/qualang_tools/results#fetching_tool) which would make this more compact and easier to write

Reply:
Yes that's fine, I can also show it to you at some point if you want :)

Reply:
yes, totally I need to get use to it for sure :)